### DISCUSSION TITLE
AARCH64 Support for Pterodactyl Wings

### DISCUSSION THREAD

**Discussion by VexoDev** [pterodactyl/panel#4530] [Feature Requests]
This is wings related,
It would be nice if the pterodactyl wings would have support for AARCH64, the ARM64 version is not working on AARCH64.

**Comment by trenutoo** [comment]
Are you referring to the Apple m1 cpu architecture, because aarch64 and arm64 mean the same things since llvm arm64 and aarch64 were merged?

Be aware that macOS by itself is not supported.

**Reply by VexoDev** [reply]
No, I'm not referring to MacBook m1, the thing - is when I try to run the wings script on my VM that is aarch64 it's not working. I know that aarch64 and arm are basically the same but it does not work on aarch64

**Reply by trenutoo** [reply]
They **are** the same nowadays. Aarch64 is the official name.

Should you have issues with running it, please use support section/discord channel and share all necessary information regarding your environment and errors.

**Reply by VexoDev** [reply]
I searched up for an answer, the only thing that I found is that it does not work or I have to change the distro from AlmaLinux to another one, the error that I got was 
exec /bin/bash: exec format error

I know that I can change the distro to another one to solve the issue but isn't it better to have support for AARCH64 by default?
